blueman/main/NetConf.py

```diff
@@ -52,7 +52,9 @@ def do_apply(self):
                 self.do_remove()
 
             ipiface = ipaddress.ip_interface('/'.join((self.netconf.ip4_address, '255.255.255.0')))
-            cmd = [have("dnsmasq"), "--port=0", "--pid-file=/var/run/dnsmasq.pan1.pid", "--except-interface=lo",
+            dnsmasq = have("dnsmasq")
+            assert dnsmasq is not None
```

Comment:
Hmm, I agree this could be a problem but I prefer to raise an exception and have it bubble up to the UI. Would you object to instead explicitly check for None and ``RuntimeError('Could not find dnsmasq')`` if not found? Same applies to dhcpcd of course.

Reply:
Fine for me. We could raise that exception from `have` and make it return non-optional `str`.